gracc-collector.spec

```diff
@@ -7,6 +7,7 @@ License:        MIT
 URL:            https://gracc.opensciencegrid.org
 Source:         %{name}.tar.gz
```

Comment:
Could you also change this to `%{name}-%{version}`?  That scheme is compatible with the new github-based fetch-sources.